pytorch/inference/docker/1.10/py3/Dockerfile.graviton.ec2.cpu

```diff
@@ -102,6 +103,7 @@ RUN conda install -c conda-forge -y \
  && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
  && pip install packaging==20.4 \
+    protobuf>=4.21.6 \
```

Comment:
Not sure if this would work - there might be some dependency conflicts. You might want to fix the version as shown in https://github.com/aws/deep-learning-containers/pull/2243/files#diff-ec22ab4008f5768525a40acc503df34dd72bce7651f65e1fa87fc999548c55afR112

Reply:
according to the testing, what is set is working as expected. 4.21.6 is compatible py3.7 and matching python apps should be working. Would rather not go backwards unless we know of an issue as the previous setting was to "be safe". the linked diff still is hard pinning down which can cause issue down the road as well, and would need to be updated. Is this what we want to do?

Reply:
Made the change. Am expecting to visit this later down the road, but would likely be updating other packages at that time.